.bootstrap.mk

```diff
@@ -50,53 +50,75 @@ SHELL       := bash
 override ROOT := $(shell path="$(CURDIR)"; while [[ "$${path}" != "/" \
  && ! -f "$${path}/.bootstrap.mk" ]]; do path="$${path%/*}"; done; echo "$${path}")
 
+## Built-in
+
 # A generic help message that parses the available targets, and lists each one
 # that has a comment on the same line with a ## prefix.
 help: ## Display this help
-	readonly pad=25; \
-	targets=(); \
-	pattern='[^:]+:[^#]*## .*'; \
-	makefiles=($(MAKEFILE_LIST)); \
-	\
-	echo; \
-	echo -e "\033[0;33mUsage:\033[0m\n\
-		make [flags...] [target...] [options...]\n\
-	\n\
-	\033[0;33mFlags:\033[0m\n\
-		See \033[1;30mmake --help\033[0m\n\
-	\n\
-	\033[0;33mTargets:\033[0m"; \
-	\
-	for mk in "$${makefiles[@]}"; do \
-		while read -r line; do \
-			if [[ "$${line}" =~ $${pattern} ]]; then \
-				targets+=("$${line}"); \
-			fi \
-		done < "$${mk}"; \
-	done; \
-	\
-	readarray -t targets < <(for t in "$${targets[@]}"; do echo "$$t"; done | sort -d); \
-	\
-	for target in "$${targets[@]}"; do \
-		printf "    \033[0;32m%-$${pad}s\033[0m %s\n" "$${target%%:*}" "$${target##*## }"; \
-	done; \
-	\
+	readonly pad=$$(printf "%0.1s" "-"{1..25})
+	print_targets() {
```

Comment:
Hey @Zebradil, thank you for your contribution, hope you are well :)

In a recent PR we introduced the trailing backslashes `\` on each line to keep make-bootstrap compatible with older make versions (see #2). In order to maintain compatibility would you mind re-adding them?

Reply:
Sure, will do.

Reply:
Done